pkg/toggle/client.go

```diff
@@ -95,12 +95,6 @@ func (c *Client) fetchEvaluation(evaluateURL string, ctx EvaluationContext) (*Re
 		return nil, err
 	}
 
-	for _, toggle := range result.Toggles {
-		if toggle.Reason == "no target matched" {
-			return nil, ErrFlagNotFound
-		}
-	}
-
```

Comment:
This was just wrong. "no target matched" doesn't mean there isn't the flag it just means that non of the configured targets matched and thus the server returned the default value.